README.md

```diff
@@ -1,25 +1,85 @@
-# PHP Version Information Page (codename `servehappy`)
-
+# WordPress PHP education initiative (codename `servehappy`).
 [![Edit in GitHub](https://img.shields.io/badge/Edit_in_GitHub--green.svg?style=social)](https://github.com/WordPress/servehappy/edit/master/README.md)
 
-**Information page about PHP versions and updates, to be used through the WordPress.org project.**
+The servehappy project is an initiative that seeks to educate WordPress site owners about the value of having the latest version of PHP powering their WordPress site.  This supports the overall goal of increasing percentage of WordPress installs running modern PHP versions.
 
-> Note: `servehappy` is only a temporary codename for now.
+## Background
 
-This is the main repository for working on the `servehappy` page(s). It is used to collect the discussions in the issue tracker and work on the page content.
+One of the core tenets of WordPress has been that of being backward compatible and the support of PHP 5.2 is and has been part and parcel of that tenet.  Over the years however, there has been increasing tension within the WordPress developer community on this support for PHP5.2 between those who support keeping the minimum PHP version requirements the same for WordPress core, and those who want to increase the minimum PHP version requirements for WordPress core.  There have been numerous arguments made for either side and you can follow some of the discussion [here](https://core.trac.wordpress.org/ticket/33381).
 
-* [Current Draft](https://github.com/WordPress/servehappy/blob/master/DRAFT.md)
+Beginning around mid 2017, there were a number of discussions initiated (in various channels) about resolving some of this tension and keeping WordPress as a project moving forward with PHP language related topics. Out of these discussions, a number of initiatives were started that support the overall _initial_  goals of:
 
-## Collecting Benefits & Stats
+- encouraging Web host providers to provide the latest PHP versions and provide an upgrade path for their customers to get on the latest versions (this is something that has been happening for a while now)
+- educating site owners about PHP and why PHP version matters.
+- decreasing the amount of reported WordPress sites running older EOL versions of PHP.
```

Comment:
It would be good to start the sentences with capitals and perhaps explain what EOL stands for.

Reply:
Done in latest commit.

Reply:
For EOL I added a brief note about it `(End of Life)` and linked to the Wikipedia entry on End of Life.  I also recognized an opportunity to link the 'PHP' text in this to the supported versions document on php.net.